src/main/routes/find-province-routes.test.ts

```diff
@@ -4,7 +4,7 @@ import app from "../config/app";
 describe("province-routes", () => {
   it("returns a 200", async () => {
     await request(app)
-      .get("/api/provinces/64f779d1-3635-41de-b02e-13f9e3956ca2")
+      .get("/api/province/64f779d1-3635-41de-b02e-13f9e3956ca2")
```

Comment:
o certo eh manter do jeito que estava, porque segue um caminho hierarquico. O que quer dizer que a pagina que estavamos antes era "provinces" e nao "province"

explicacao:

http://www.mywebsite.com/provinces/quebec:
This format follows a hierarchical structure, where "provinces" is the parent directory or route, and "quebec" is a child or subdirectory representing the specific province. It's a clean and intuitive way to organize your website's URLs.
o mesmo se aplica pra cities

Reply:
Valeu... Vou fechar esse PR então.